ios/XMTPModule.swift

```diff
@@ -47,16 +47,32 @@ public class XMTPModule: Module {
 	actor ClientsManager {
 		private var clients: [String: XMTP.Client] = [:]
 
-		// A method to update the conversations
+		// A method to update the client
 		func updateClient(key: String, client: XMTP.Client) {
 			ContentJson.initCodecs(client: client)
 			clients[key] = client
 		}
 
-		// A method to retrieve a conversation
+		// A method to retrieve a client
 		func getClient(key: String) -> XMTP.Client? {
 			return clients[key]
 		}
+        
+		// A method to disconnect all dbs
+		func dropAllLocalDatabaseConnections() throws {
+			for (_, client) in clients {
+				// Call the method on each client
+				try client.dropLocalDatabaseConnection()
+			}
+		}
+
+		// A method to reconnect all dbs
+		func reconnectAllLocalDatabaseConnections() async throws {
+			for (_, client) in clients {
+				// Call the method on each client
+				try await client.reconnectLocalDatabase()
```

Comment:
This is a blocking call. Hopefully this won't create too much lag when people reopen the app if they have a lot of clients. I really think we should consider setting a limit of how many clients a user can be logged in with. For MVP I really think we should just allow 1.

Reply:
This is a blocking call but in an actor, isn't everything called in an async manner? I'm not used to actors